public/css/product-promos.css

```diff
@@ -0,0 +1,171 @@
+.product-promo {
+  position: relative;
+  overflow: hidden;
+  border-radius: 16px;
+  padding: 32px 42px;
+  color: rgba(255, 255, 255, 1);
```

Comment:
Could this be a variable?

Reply:
Could be/should- but then we'd want to change it everywhere.